packages/file-type-icons/src/FileTypeIconMap.ts

```diff
@@ -83,6 +83,7 @@ export const FileTypeIconMap: { [key: string]: { extensions?: string[] } } = {
       'csproj',
       'css',
       'ctp',
+      'cvs',
```

Comment:
I think this should be mapped to vector file type. It's tied to some a discontinued illustration app according to Carlos.